### PR TITLE
B 26

### PR DESCRIPTION
**Pull request by dimon1976**
metods registration, servlet, jsp and small refactor

### CODE REVIEW

File: src/main/java/by/fakeonliner/repository/ShopRepository.java [hunk 1]
```diff
@@ -5,32 +5,64 @@
 import java.util.ArrayList;
 import java.util.List;
 
-public class ShopRepository implements ShopDao{
+public class ShopRepository implements ShopDao {
 
     private final List<Shop> shops = new ArrayList<>();
+    private long id = 0;
```
Comment (simonpirko) [inline src/main/java/by/fakeonliner/repository/ShopRepository.java:11]: What for???

File: src/main/java/by/fakeonliner/service/ShopService.java [hunk 2]
```diff
@@ -1,4 +1,17 @@
 package by.fakeonliner.service;
 
+import by.fakeonliner.entity.Shop;
+import by.fakeonliner.repository.ShopRepository;
+
 public class ShopService {
+    private ShopRepository shopRepository = new ShopRepository();
```
Comment (simonpirko) [inline src/main/java/by/fakeonliner/service/ShopService.java]: What use concrete class? May be make general interface

**Review by simonpirko — APPROVED** [review]

File: src/main/java/by/fakeonliner/repository/ShopRepository.java (hunk 1 above)
Comment (dimon1976) [reply src/main/java/by/fakeonliner/repository/ShopRepository.java:11]: increment id Shop

File: src/main/java/by/fakeonliner/service/ShopService.java (hunk 2 above)
Comment (dimon1976) [reply src/main/java/by/fakeonliner/service/ShopService.java]: fix

File: src/main/java/by/fakeonliner/service/ShopService.java [hunk 3]
```diff
@@ -1,4 +1,18 @@
 package by.fakeonliner.service;
 
+import by.fakeonliner.entity.Shop;
+import by.fakeonliner.repository.ShopDao;
+import by.fakeonliner.repository.ShopRepository;
+
 public class ShopService {
+    private final ShopDao shopRepository = new ShopRepository();
```
Comment (simonpirko) [inline src/main/java/by/fakeonliner/service/ShopService.java]: Dao or repository?

**Review by simonpirko — APPROVED** [review]

File: src/main/java/by/fakeonliner/service/ShopService.java (hunk 3 above)
Comment (dimon1976) [reply src/main/java/by/fakeonliner/service/ShopService.java]: DAO fix

**Pull request closed without merging**